### ISSUE TITLE
JAWS unable to navigate and read headers and columns for DetailsList

### ISSUE THREAD

**Issue by tpanpalia** [OfficeDev/office-ui-fabric-react#7438]
For DetailsList, JAWS is unable to navigate and read the headers and columns in the grid when user navigates using right arrow.

**Comment by ecraig12345** [comment]
Hi @tpanpalia, would you mind providing a link to a page where this issue can be reproduced? Otherwise it's much harder to debug.

**Comment by ecraig12345** [comment]
@tpanpalia Could you please see if the same issue reproduces on the [Fabric website](https://fabricweb.z5.web.core.windows.net/oufr/6.121.0/#/examples/detailslist/basic)? Or if it doesn't repro there, it would help if you can get a simple repro in a [codepen](http://aka.ms/fabricpen). (Sorry about not being more specific about that previously.)

**Comment by ecraig12345** [comment]
@KevinTCoughlin David thought I should bring this to your attention since you've worked a lot on DetailsList. Does this sound familiar?

**Comment by tpanpalia** [comment]
Behavior without JAWS - 
Right arrow key to navigate through different grid headers, down arrow to navigate to each row and right arrow key to navigate through accessible column elements.

Behavior with JAWS - 
Following the above steps (Behavior without JAWS), JAWS is unable to read.
Note : Navigation through down arrow key, JAWS is able to read.

We observed the same behavior in https://developer.microsoft.com/en-us/fabric#/components/detailslist and https://fabricweb.z5.web.core.windows.net/oufr/6.121.0/#/examples/detailslist/basic and in our application also https://eccorevluumuipreproddevatm.trafficmanager.net:20200/.

**Comment by KevinTCoughlin** [comment]
I'm unsure what the difference between JAWS and Narrator are here.

Currently, DetailsList's column headers are read aloud by Narrator in Edge:

```shell
Column operations for File type, Press to sort on File type, 
Name, button,, Column Header, Column <say-as interpret-as="cardinal">2</say-as> of <say-as interpret-as="cardinal">5</say-as> 
Date Modified, button,, Column Header, Column <say-as interpret-as="cardinal">3</say-as> of <say-as interpret-as="cardinal">5</say-as> 
Modified By, button,, Column Header, Column <say-as interpret-as="cardinal">4</say-as> of <say-as interpret-as="cardinal">5</say-as> 
File Size, button,, Column Header, Column <say-as interpret-as="cardinal">5</say-as> of <say-as interpret-as="cardinal">5</say-as> 
accdb file iconLoremipsum.xsn 8/26/2012 Lorem Ipsum 115 KB, non-selected, 
```

And DetailsList's rows including grid cell content are read aloud by Narrator in Edge when using `UP/DOWN`:

```shell
accdb file iconLoremipsum.xsn 8/26/2012 Lorem Ipsum 115 KB, non-selected, 
```

Regarding the following:
>right arrow key to navigate through accessible column elements

As far as I know, granular selection of cell contents isn't possible. But the entire row's contents are read aloud in Narrator. So I'm unsure of our commitment to support this scenario.

I don't currently have JAWS setup on my machine to debug further, but that would be next step for investigation.

@tpanpalia does this repro in JAWS across all major browsers?

**Comment by ecraig12345** [comment]
Reassigning to Kevin per David's suggestion.

**Comment by tpanpalia** [comment]
JAWS is more compatible with IE. 
However, we have verified the behavior in IE, Chrome and Edge browsers. This repro in all the browsers. JAWS is able to read entire row content (using up/down arrow key), but, for Edge, the text is not highlighted for the columns.

**Comment by betrue-final-final** [comment]
This sounds like an Edge bug.

**Comment by tpanpalia** [comment]
Is there any possibility to support JAWS behaviour with Right arrow key to navigate through different grid headers, down arrow to navigate to each row and right arrow key to navigate through accessible column elements?

**Comment by dzearing** [comment]
@KevinTCoughlin this one also never got traction

**Comment by khmakoto** [comment]
@tpanpalia is this still reproducing for you? If it is, are you able to reproduce it with Edge+Narrator or does it only happen in JAWS? Thanks for the engagement and sorry for the long time it has taken to follow up on this issue!
